### PR TITLE
Add actuator and enable metrics, add Grafana configs

### LINKED ISSUE #6520: [Feature]  configration of grafana
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

In order to observe the operation of the DS system, I think it is very necessary to use the grafana plug-in.
we can see the jvm running state on grafana, and i think some indicators can also display on grafana.
eg:
count of task in a period time (10 mins): 
"select count(*) from t_ds_task_instance where start_time between $startDate and $endDate";

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kezhenxu94**
Also closes #6520 

PTAL @CalvinKirs @dailidong @lenboo @caishunfeng

### CODE REVIEW

**Review by CalvinKirs — APPROVED** [review]
LGTM

File: install.sh [hunk 1]
```diff
@@ -21,72 +21,16 @@ workDir=`cd ${workDir};pwd`
...
-sed -i ${txt} "s@^#*worker.tenant.auto.create=.*@worker.tenant.auto.create=${workerTenantAutoCreate}@g" conf/worker.properties
-sed -i ${txt} "s@^#*alert.listen.host=.*@alert.listen.host=${alertServer}@g" conf/worker.properties
-
-# Change configuration in conf/application-api.properties
-sed -i ${txt} "s@^#*server.port=.*@server.port=${apiServerPort}@g" conf/application-api.properties
-
-# Change configuration in conf/registry.properties
-sed -i ${txt} "s@^#*registry.plugin.dir=.*@registry.plugin.dir=${installPath}/${registryPluginDir}@g" conf/registry.properties
-sed -i ${txt} "s@^#*registry.plugin.name=.*@registry.plugin.name=${registryPluginName}@g" conf/registry.properties
-sed -i ${txt} "s@^#*registry.servers=.*@registry.servers=${registryServers}@g" conf/registry.properties
```
Comment (zhongjiajie) [inline install.sh:78]: Hi @kezhenxu94, why should we remove those lines?

**Comment by chengshiwen** [comment]
This pr violates the principle of minimum code submission, introduces irrelevant modifications and deletes template files related to docker, resulting in docker and k8s not working properly. 
image

**Comment by chengshiwen** [comment]
Maybe related issue: #6998, #6994, #6885
@CalvinKirs @kezhenxu94

**Comment by kezhenxu94** [comment]
> Maybe related issue: #6998, #6994, #6885
> 
> @CalvinKirs @kezhenxu94 

Only #6998 is related to this as this only lands in dev branch and the other two issues are 2.0.0 and 1.3.9. At this moment dev is not expected to be used in production environment

**Pull request merged**